File: JavaScript/프로그래머스/0/181927. 마지막 두 원소/마지막 두 원소.js
function solution(num_list) {
    var last = num_list.length; 
    var answer = Array(last+1);
    var n=0;
    for(var i of num_list){
        answer[n]=i;
        n++;
    }
    if(num_list[last-1]>num_list[last-2]){
        answer[last]=num_list[last-1]-num_list[last-2];
    }
    else {
        answer[last]=num_list[last-1]*2;
    }
    return answer;
}

File: 프로그래머스/0/181927. 마지막 두 원소/마지막 두 원소.cs
using System;

public class Solution {
    public int[] solution(int[] num_list) {
        int[] answer = new int[num_list.Length+1];
        for(int i=0; i<num_list.Length;i++){
            answer[i]=num_list[i];
        }
        if(num_list[num_list.Length-1]>num_list[num_list.Length-2])
            answer[answer.Length-1]=num_list[num_list.Length-1]-num_list[num_list.Length-2];
        else answer[answer.Length-1]=num_list[num_list.Length-1]*2;
        return answer;
    }
}